src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -4466,6 +4497,157 @@ async def _cleanup():
+                if agent is not None:
+                    session = GatewaySession.from_dict(
+                        data, self.config.session_config.max_messages,
+                    )
+                    session._is_active = True
+                    self._sessions[sid] = session
+                    if not session._inbox.empty():
+                        if not session._is_executing:
+                            session.mark_executing(True)
+                        asyncio.create_task(
+                            self._run_session_queue(session, agent, session.client_id)
+                        )
```

Ruff (0.15.21)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)